Tasks/KubernetesV1/_buildConfigs/Node20/package.json

```diff
@@ -7,7 +7,7 @@
     "@types/q": "^1.5.0",
     "@types/uuid": "^8.3.0",
     "agent-base": "^6.0.2",
-    "azure-pipelines-task-lib": "^4.13.0",
+    "azure-pipelines-task-lib": "^4.11.0",
```

Comment:
Why are we downgrading to older version?

Reply:
upgrade to 4.13.0. The reason I downgraded becauses I was changing it to 4.11.0 for all tasks. So didnt't notice that KubernetesV1 had 4.13.0. So reverted the change now.